mico-core/src/main/java/io/github/ust/mico/core/util/KubernetesNameNormalizer.java

```diff
@@ -37,6 +37,7 @@
     private final static String REGEX_MULTIPLE_DASHES = "[-]+";
     private final static String REGEX_FIRST_OR_LAST_CHAR_IS_A_DASH = "^-|-$";
     private final static String REGEX_MATCH_VALID_FIRST_CHAR = "^[a-z]+.*";
+    public final static int MAX_LABLE_SIZE = 54;
```

Comment:
Fix typo → `MAX_LABEL_SIZE`.
However I would prefer a different name like `MICO_NAME_MAX_SIZE`.

Reply:
Add a JavaDoc to this variable and explain, why it is 54. Example:
A max limit of the MICO names (`{@link MicoApplication}`, `{@link MicoService}` and `{@link MicoServiceInterface}`) is required because they are used as values of Kubernetes labels that have a limit of 63. Furthermore the name is used to create a UID that adds 9 characters to it. Therefore the limit have to be set to 54.